frontend/src/core/cells/cells.ts

```diff
@@ -231,6 +231,25 @@ const {
       scrollKey: null,
     };
   },
+  focusCellAtDefinition: (
```

Comment:
Even though its kinda similar to `focusCell`, I think we can make this a function outside of the reducer in `core/codemirror/go-to-definition`. You can access the store by `store.get(atomName)`.